recipes-bsp/efitools/efitools_%.bbappend

```diff
@@ -0,0 +1,7 @@
+# Remove patch if using zeus
+SRC_URI_remove = "file://0001-console.c-Fix-compilation-against-latest-usr-include.patch"
```

Comment:
@Asiderr We could have a single .inc file with a name indicating it's purpose (like efitools-zeus-compilation.inc) and include the common part in both recipes.

Reply:
Fixed.